### PR TITLE
Added support for graphql-orders.

### PR DESCRIPTION
**Pull request by dmitrijs-voronovs**
Added support for graphql-orders.

### CODE REVIEW

File: src/Model/Resolver/ProductResolver.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        $this->productRepository = $productRepository;
+    }
+
+    /**
+     * Get All Product Items of Order.
+     * @inheritdoc
+     */
+    public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
+    {
+        if (!isset($value['products'])) {
```
Comment (eli-l) [inline src/Model/Resolver/ProductResolver.php]: We can avoid this check if we set this argument as non-nullable in graphql schema.

File: src/Model/Resolver/ExpandedOrderResolver.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        $itemsData = [];
+        $trackNumbers = [];
+
+        $customerId = $context->getUserId();
+        $this->checkCustomerAccount->execute($customerId, $context->getUserType());
+
+        $orderId = $args['id'];
+        $order = $this->orderRepository->get($orderId);
+
+        if ($customerId != $order->getCustomerId()) {
```
[the remark this answers is not in the document]
Comment (dmitrijs-voronovs) [reply src/Model/Resolver/ExpandedOrderResolver.php]: Yes, it is.

File: src/Model/Resolver/ProductResolver.php (hunk 1 above)
Comment (dmitrijs-voronovs) [reply src/Model/Resolver/ProductResolver.php]: Can you, please, show me, from where I can get access to this argument.
Is it ```order_products: [ProductInterface] @resolver(class: "\\ScandiPWA\\QuoteGraphQl\\Model\\Resolver\\ProductResolver")```
where I should replace `[ProductInterface]` with `[ProductInterface!]` ?

Comment (eli-l) [reply src/Model/Resolver/ProductResolver.php]: My mistake. Let's leave as is.

**Pull request merged**